void
testTry3
(
)
{
int
a
=
0
;
int
b
=
0
;
try
{
a
=
1
;
}
catch
(
ArithmeticException
e
)
{
b
=
1
;
}
catch
(
RuntimeException
e
)
{
}
if
(
a
==
0
)
{
}
if
(
b
==
0
)
{
}
}
